scheduler/kubernetes/install.md

```diff
@@ -124,14 +124,7 @@ $ kubectl label nodes minion5 "px/enabled=false" --overwrite
 Portworx is deployed as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). Therefore it automatically scales as you grow your Kubernetes cluster.  There are no additional requirements to install Portworx on the new nodes in your Kubernetes cluster.
 
 #### Installing behind the HTTP proxy
```

Comment:
Need to maintain the existing k8s spec generation example here. The redirect to the common page ` [Passing environment variables](/runc/options.html#environment-variables` is too generic and k8s users will get confused.

Reply:
Ok after talking to Harsh it makes sense to move the curl example for install.portworx.com back to the kube install pages.    Its moved back to cmdargs.md.   The link for passing env arguments is still there with the example.